test_runner/src/main/kotlin/ftl/config/Device.kt

```diff
@@ -19,3 +23,19 @@ data class Device(
           orientation: $orientation""".trimStartLine()
     }
```

Comment:
I know you don't change this line but:
```
override fun toString(): String {
        return """
        - model: $model
          version: $version
          locale: $locale
          orientation: $orientation""".trimStartLine()
    }
```
vs

```
override fun toString() = """
        - model: $model
          version: $version
          locale: $locale
          orientation: $orientation""".trimStartLine()
```
This only for discussion to keep one style. Personally I prefer second.